examples/arduino-uno/src/bin/uno-eeprom.rs

```diff
@@ -0,0 +1,36 @@
+/*!
+ * Demonstration of writing to and reading from the eeprom.
+ */
+#![no_std]
+#![no_main]
+
+use arduino_hal::prelude::*;
+use panic_halt as _;
+
+// use embedded_storage::nor_flash::ReadNorFlash;
+
+#[arduino_hal::entry]
+fn main() -> ! {
+    let dp = arduino_hal::Peripherals::take().unwrap();
+    let pins = arduino_hal::pins!(dp);
+    let mut serial = arduino_hal::default_serial!(dp, pins, 57600);
+
+    let mut ep = arduino_hal::Eeprom::new(dp.EEPROM);
+    let ep_capacity = ep.capacity();
+    ufmt::uwriteln!(&mut serial, "eeprom capacity is:{}\r", ep_capacity).void_unwrap();
+
+    // atmega328p eeprom size is 1024
+    let mut data = [0_u8; 1024];
```

Comment:
... and exposing the capacity constant becomes useful here where you can now write this instead:

```rust
let mut data = [0u8; arduino_hal::Eeprom::CAPACITY];
```

Reply:
review fixed